deployments/base/usdbc/configuration.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "Compound USDbC",
+  "symbol": "cUSDbCv3",
+  "baseToken": "USDbC",
+  "baseTokenAddress": "0xd9aAEc86B65D86f6A7B5B1b0c42FFA531710b6CA",
+  "baseTokenPriceFeed": "0x7e860098F58bBFC8648a4311b374B1D669a2bc6B",
```

Comment:
[ChainLink docs](https://docs.chain.link/data-feeds/price-feeds/addresses?network=base) don't actually list mainnet Base feeds yet, but I can confirm that this is an `EACAggregatorProxy` contract that returns a 6 decimal number that is just shy of 1